src/packaging/packages.targets

```diff
@@ -1,37 +1,18 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-    <UsingTask TaskName="GetBuildUid" TaskFactory="CodeTaskFactory" AssemblyName="Microsoft.Build.Tasks.v12.0">
-        <ParameterGroup>
-            <BuildUid ParameterType="System.String" Output="true" />
-        </ParameterGroup>
-        <Task>
-            <Code>
-                BuildUid = System.Guid.NewGuid().ToString().Replace("-", "").Substring(0, 8);
```

Comment:
The `BuildUid` is essential in cheating the nuget cache, so for each build a unique package name is applied. This is because a machine could be running multiple builds, esp. CI machines and if they all produced the same package names then nuget restore would no-op and use bits from the nuget cache when we issue publish etc.